appeals/web/src/server/lib/mappers/__tests__/mappers.test.js

```diff
@@ -50,7 +50,7 @@ describe('appeal-mapper', () => {
 			const preFormattedDate = '2023-10-11T01:00:00.000Z';
 			const mappedDateHtml =
```

Comment:
Probably need to rename this too then as it's a text.

Reply:
Done: https://github.com/Planning-Inspectorate/appeals-back-office/pull/519/commits/a78ff2be879d259f84c6a4ebe7273023e59389ee